### PR TITLE
PageIntro title postscript

### PR DESCRIPTION
**Pull request by dlrice**
## Purpose
In the implementation of peptide search results the need has arisen to supply a postscript to the `PageIntro`'s title.

## Approach
Create optional `titlePostcript` prop

## Testing
Unit tests

## Stories
Page intro with knobs

## Checklist
- [x] My PR is scoped properly, and “does one thing only”
- [x] I have reviewed my own code
- [x] I have checked that linting checks pass and type safety is respected
- [x] I have checked that tests pass and coverage has at least improved, and if not explained the reasons why
- [x] Were all the edited/created files in TypeScript? If not, please list the files you left in JavaScript and the reason to not update them.
- [x] For the stories you created/updated, are the props modifiables through knobs?

### CODE REVIEW

File: src/components/page-intro.tsx [hunk 1]
```diff
@@ -12,9 +11,39 @@ import '../styles/components/page-intro.scss';
...
+  resultsCount = 0,
+  children,
+  links,
+  showContent = false,
+  titlePostscript,
+}) => {
+  const [displayContent, setDisplayContent] = useState(showContent);
   return (
     <div className="page-intro">
       <h2>
```
Comment (aurel-l) [inline src/components/page-intro.tsx:49]: This should default to `h1`, and be configurable through props (see `headingLevel` prop in `HeroContainer` or `Tile` for example) but should have the `.big` classname to define its size regardless of the heading level

File: src/components/page-intro.tsx [hunk 2]
```diff
@@ -12,9 +11,39 @@ import '../styles/components/page-intro.scss';
...
+   */
+  resultsCount?: number;
+  /**
+   * Content revealed on toggle
+   */
+  children?: ReactNode;
+  /**
+   * Links revealed on toggle
+   */
+  links?: { title: string; destination: string }[];
```
[the remark this answers is not in the document]
Comment (aurel-l) [reply src/components/page-intro.tsx]: not relevant anymore

File: src/components/page-intro.tsx (hunk 1 above)
Comment (dlrice) [reply src/components/page-intro.tsx]: Good idea - updated

Comment (dlrice) [reply src/components/page-intro.tsx]: forgot to do the `.big` aspect. doing now

**Review by aurel-l — APPROVED** [review]
Looks good 👍🏽 
In the story the postscript looks bigger than the count, but I guess it's just because it's passing plain text as a prop, I guess it's the responsability of the user of the component to style the postscript however need they need, right?

File: src/components/page-intro.tsx (hunk 1 above)
Comment (aurel-l) [reply src/components/page-intro.tsx]: Oh, right, forgot about that in my review now

**Comment by dlrice** [comment]
> Looks good 👍🏽
> In the story the postscript looks bigger than the count, but I guess it's just because it's passing plain text as a prop, I guess it's the responsability of the user of the component to style the postscript however need they need, right?

Yeah - just left it flexible for the user to decide

**Review by aurel-l — APPROVED** [review]

**Pull request merged**